src/main/scala/mesosphere/marathon/storage/migration/Migration.scala

```diff
@@ -223,24 +238,13 @@ object StorageVersions {
       .build()
   }
 
-  def current: StorageVersion = {
-    BuildInfo.version match {
-      case VersionRegex(major, minor, patch) =>
-        StorageVersions(
-          major.toInt,
-          minor.toInt,
-          patch.toInt,
-          StorageVersion.StorageFormat.PERSISTENCE_STORE
-        )
-      case BuildInfo.DefaultBuildVersion =>
-        StorageVersions(
-          BuildInfo.DefaultMajor,
-          BuildInfo.DefaultMinor,
-          BuildInfo.DefaultPatch,
-          StorageVersion.StorageFormat.PERSISTENCE_STORE
-        )
-    }
-  }
+  /**
+    * Get the migration target version from a list of migration steps.
+    *
+    * @param steps The steps of a a migration.
+    * @return The target version of the migration steps.
+    */
+  def apply(steps: List[MigrationAction]): StorageVersion = steps.map(action => action._1).max
```

Comment:
sorry, I just don't like _1, if it makes sense to you can we move to something more readable?